xpdschema/schemas/run_start.json

```diff
@@ -0,0 +1,178 @@
+{
+  "type": "object",
+  "properties":{
+    "facility": {
+      "type": "string",
+      "description": "Facility Name (NSLS-II, APS, etc)"
+    },
+    "bt_piLast":{
+      "type": "string",
+      "description": "Last name of PI"
+    },
+    "bt_wavelength":{
+      "type": "number",
+      "description": "x-ray wavelength in Angstrom, Depreciation Warning: this will depreciated in upcoming releases, please do not use this going forward"
+    },
+    "bt_safN":{
+      "type": "string",
+      "description": "Safety Approval Number of beamtime"
+    },
+    "bt_experimenters":{
+      "type": "array",
+      "description": "list of experimenter names in [first, last,...] format"
+    },
+    "bt_uid":{
+      "type": "string",
+      "description": "auto-generated uid for Beamtime object"
```

Comment:
do we want to say beamtime "object" or just beamtime
How about this:
"auto-generated uid for the current Beamtime. All scans collected during this beamtime will be tagged with this uid."